README.md

```diff
@@ -1,20 +1,116 @@
+    <img align='center' src="https://img.shields.io/badge/LICENSE-GPL-green"/>
+  </a>
+  <a href="https://github.com/Ashrockzzz2003/Data_Structures_and_Algorithms" target="_blank" rel="noreferrer">
+    <img align='center' src="https://img.shields.io/github/created-at/Ashrockzzz2003/Data_Structures_and_Algorithms"/>
+  </a>  
+</div>
 
-<!--
-## Contributions required on
+---
 
-- [ ] `Cycle Detection with BFS`
-- [ ] `Printing the cycle found with BFS`
-- [ ] `Printing the cycle found with DFS`
-- [ ] `Testing a graph for bipartiteness`
-- [ ] `Optimizing Dijkstra Code`-->
+A collection of data structures and algorithms implemented in Python. Inspired by the **Design and Analysis of Algorithms** course from the 3rd year of **B. Tech, CSE** at **Amrita Vishwa Vidyapeetham, Coimbatore**, taught by **Dr. Vidya Balasubramanian** ma'am.  
 
-## Contributions
+---
 
-| Contribution | Contributor |
-| ------------ | ----------- |
-| `.py` | [`Ashwin Narayanan S`](https://ashrockzzz2003.github.io/portfolio/) |
-| `.cpp` | [`Aasil`](https://github.com/mdxaasil) |
+## **Table of Contents**
+
+1. [Getting Started](#getting-started)
+   - [Clone the Repository](#clone-the-repository)
+2. [Running Code](#running-code)
+   - [Run Python Code](#run-python-code)
+   - [Run C++ Code](#run-c-code)
+   - [Run Java Code](#run-java-code)
+   - [Run Go Code](#run-go-code)
+3. [Contributing](#contributing)
+4. [Contributors](#contributors)
+5. [License](#license)
+6. [Stay Connected](#stay-connected)
+
+---
+
+## **Getting Started**
+
+### **Clone the Repository**
+
+Getting started is easy! Just follow these steps to bring the repository to your local machine:
+
+```bash
+git clone https://github.com/Ashrockzzz2003/Data_Structures_and_Algorithms.git  
+cd Data_Structures_and_Algorithms
+```
+
+Now you’re ready to explore and experiment with the algorithms in this repository.
+
+---
+
+## **Running Code**
+
+### **Run Python Code**
+
+Execute Python scripts directly in your terminal:
+
+```bash
+python sorting/quick_sort.py
+```
+
+### **Run C++ Code**
+
+To compile and run C++ files:
+
+```bash
+g++ -o quick_sort sorting/quick_sort.cpp
+./quick_sort
+```
+
+### **Run Java Code**
+
+For Java programs, first compile and then run:
+
+```bash
+javac sorting/QuickSort.java
+java sorting/QuickSort
+```
+
+### **Run Go Code**
+
+Run Go programs using the `go run` command:
+
+```bash
+go run sorting/quick_sort.go
+```
+
+---
+
+## **Contributing**
+
+We’re excited to have you contribute! Whether it’s adding a new algorithm, fixing a bug, or improving documentation, every contribution counts. For detailed instructions, check out the [CONTRIBUTING.md](docs/CONTRIBUTING.md) file.
+
+---
+
+## **Contributors**
+
+| **Contribution Type**    | **Contributor**      |
+| ------------------------ | -------------------- |
+| Python Implementation   | Ashwin Narayanan S   |
+| C++ Implementation      | Aasil                |
+
```

Comment:
The word `implementation` sounds odd.
Also, probably link `Contributor` names to GitHub account URL.

Reply:
Will do that anna @Ashrockzzz2003